Source/Scene/ModelExperimental/ModelExperimentalAnimationChannel.js

```diff
@@ -275,17 +276,17 @@ ModelExperimentalAnimationChannel.prototype.animate = function (time) {
         : spline.wrapTime(time);
       morphWeights[i] = spline.evaluate(localAnimationTime);
     }
+  } else if (runtimeNode.userAnimated) {
+    // If the node is being animated externally, ignore the glTF animation.
+    return;
```

Comment:
Just to clarify, the weights still need to be updated even if the user modified the nodes?

Reply:
Yes, I believe the animation of morph weights in `Model` happens whether or not the node's transform has been modified by the user. The `translation`, `rotation`, and `scale` properties are ignored, but the weights are unrelated.